pkg/tracing/populate_span_transformer_test.go

```diff
@@ -46,7 +46,13 @@ func TestPopulateSpan(t *testing.T) {
 	wantEvent.SetType("hello.world")
 	wantEvent.SetSource("example.com")
 
+	destination := "some-url"
+
 	expectedAttributes := map[string]interface{}{
+		"messaging.system":        "knative",
+		"messaging.protocol":      "HTTP",
```

Comment:
I see that `http` is the protocol used, do we support https? How about protocol version maybe we should be explicit as there is wip (if I understood correctly) for [http/2](https://github.com/knative/eventing/issues/3312)?

Reply:
Hey @skonto
If I understood correctly, once the http/2 feature is wrapped up, the protocol string could change and hence hardcoding a value in the test is not a good idea.

The currently implementation has it hardcoded [here](https://github.com/knative/eventing/blob/master/pkg/tracing/attributes.go#L35) which would need to change once the http/2 feature is completed. 

If that change is imminent;
- the assertion could be updated to match a regex 
- the string could be pulled out and put into an enum ( would still  only be configured for `HTTP` for now )
- and the test assertion for protocol isn't very valuable the assertion for protocol could be dropped

wdyt ?